docker/control/bashrc

```diff
@@ -0,0 +1,14 @@
+cat <<EOF 
+Welcome to Jepsen on Docker
+===========================
+
+This container runs the Jepsen tests in sub-containers.
+
+You are currently in the base dir of the git repo for Jepsen.
+If you modify the core jepsen library make sure you "lein install" it so other tests can access.
+
+To run a test:
+   cd mongodb && lein run
```

Comment:
The reason why I mentioned `mongodb` here is mainly because its README.md is well-documented than others now